src/cassandra.go

```diff
@@ -103,6 +101,74 @@ func main() {
 	fatalIfErr(i.Publish())
 }
 
+func runMetricCollection(i *integration.Integration, e *integration.Entity, jmxClient *gojmx.Client) error {
+	if !args.LongRunning {
+		return collectMetrics(e, jmxClient)
+	}
+
+	heartBeat := time.NewTicker(time.Duration(args.HeartbeatInterval) * time.Second)
+	metricInterval := time.NewTicker(time.Millisecond)
+
+	go func() {
+		for range heartBeat.C {
+			log.Debug("Sending heartBeat")
+			// heartbeat signal for long-running integrations
+			// https://docs.newrelic.com/docs/integrations/integrations-sdk/file-specifications/host-integrations-newer-configuration-format#timeout
+			fmt.Println("{}")
+		}
+	}()
+
+	// Force the first collection to happen immediately.
+	first := true
+
+	for range metricInterval.C {
```

Comment:
Maybe I'm laking some context here and this situation just can't happen but I'm asking just in case: as long as long-running-mode blocks here, if [inventory](https://github.com/newrelic/nri-cassandra/pull/99/files#diff-1ebfe0c709ba4698e9e87718c131004c9654574a1d3882080211984533bc53e0L96) was also required  it wouldn't get executed, right?

Reply:
I guess it will retrieved as before by a separate instance. Since no connection is required it will be scraped multiple times

Reply:
yes, Inventory is collected by a separate instance